ethereum/spec/src/main/java/tech/pegasys/teku/spec/datastructures/blocks/blockbody/versions/phase0/BeaconBlockBodyBuilderPhase0.java

```diff
@@ -46,6 +46,14 @@ public class BeaconBlockBodyBuilderPhase0 implements BeaconBlockBodyBuilder {
   protected SszList<Deposit> deposits;
   protected SszList<SignedVoluntaryExit> voluntaryExits;
 
+  public BeaconBlockBodyBuilderPhase0() {
```

Comment:
i think we should initialise to null and just call it if we're making it not null, and then this fn shouldn't need to exist maybe?